### DISCUSSION TITLE
Dotnet new sem o main.

### DISCUSSION THREAD

**Discussion by casadocaio** [balta-io/forum#508] [Tira Dúvidas] [answered]
Boa tarde. Muito obrigado pelo conteúdo.

Gostaria de saber por que quando inicio o projeto pelo dotnet new console ele cria o csproj, o program.cs mas o program.cs já vem só com um hello world, e não vem com using, namespace e nem o main, estou tendo que criá-los na mão, tem alguma configuração que está fazendo isso?

**Comment by andrebaltieri** [comment, marked as answer]
Duplicado
https://github.com/balta-io/forum/discussions/366#discussioncomment-6516324
